doc/nrf/libraries/networking/azure_iot_hub.rst

```diff
@@ -89,7 +216,7 @@ Configuration without using DPS
 
 To connect to Azure IoT Hub without using DPS, complete the following minimum required configuration:
 
-1. In the `Azure Portal`_, navigate to :guilabel:`IoT Hub` and select the desired IoT hub.
+1. In the `Azure Portal`_, navigate to :guilabel:`IoT Hub` and select the desired IoT Hub.
```

Comment:
```suggestion
1. To retrieve your IoT Hub hostname, run the following command:
    .. code-block:: console

        az iot hub show --name <hub_name> --query "properties.hostName"

#. Configure :kconfig:option:`CONFIG_AZURE_IOT_HUB_HOSTNAME` to the returned address.
```

Wasn't able to remove the next line because it was not changed in the PR, but it should be removed after this change.

Reply:
Thanks! Applied this and the other suggestions, and removed the line below